protocol/da-footprint-block-limit.md

```diff
@@ -0,0 +1,299 @@
+have limited that block's transactions to a smaller DA footprint. For those blocks, we also looked at the
+distribution of the excess DA footprint as an indicator of how much those blocks would have been limited.
+
+The following tables show, for each analyzed chain, the resulting statistics.
+* *Scalar*: DA footprint gas scalar value.
+* *Effective Limit*: The DA usage limit that the given gas scalar would imply (`block_gas_limit / da_footprint_gas_scalar`), in estimated compressed bytes.
+* *Exceeding Limit*: Number and percent of blocks for wich the total DA footprint exceeds the gas limit. Those blocks
+  would have been limited to include fewer transactions.
+* *Exc. Gas Usage*: Number and percent of blocks for wich the total DA footprint exceeds the total gas usage. For those blocks, the `block.gas_used`
+  field would equal the total block DA footprint instead of the total transaction gas used.
+* *Avg. Utilization*: Average utilization of DA footprint resource.
+* *Max Utilization*: Maximum utilization of DA footprint resource.
+
+#### OP Mainnet
+
+**[Random Sampling](https://htmlpreview.github.io/?https://github.com/ethereum-optimism/op-analytics/blob/main/notebooks/adhoc/jovian_analysis/notebooks/saved_output_html/op_random_2025-04-29_2025-08-26.html):**
+_Sample Size:_ `51,840`
+
+| Scalar | Effective Limit | Exceeding Limit | Exc. Gas Usage | Avg. Utilization | Max Utilization |
+| -------|-----------------|-----------------|----------------|------------------|-----------------|
+| 160    | 250,000         | 0               | 0              | 1.1%             | 44.8%           |
+| 400    | 100,000         | 1 (0.00%)       | 3 (0.01%)      | 2.8%             | 112.1%          |
+| 600    |  66,666         | 2 (0.00%)       | 17 (0.03%)     | 4.1%             | 168.2%          |
+| 800    |  50,000         | 10 (0.02%)      | 47 (0.09%)     | 5.5%             | 224.2%          |
+
+![OP Mainnet (random) DA size estimation distribution](./assets/da-footprint-block-limit/OPM-random-DA-distribution.png)
+
+**[Top 1 Percentile Calldata Usage](https://htmlpreview.github.io/?https://github.com/ethereum-optimism/op-analytics/blob/main/notebooks/adhoc/jovian_analysis/notebooks/saved_output_html/op_top_percentile_2025-04-29_2025-08-26.html):**
+_Sample Size:_ `52,165`
+
+| Scalar | Effective Limit | Exceeding Limit | Exc. Gas Usage | Avg. Utilization | Max Utilization |
+|--------|-----------------|-----------------|----------------|------------------|-----------------|
+| 160    | 250,000         | 0               | 0              | 1.1%             | 40.0%           |
+| 400    | 100,000         | 0               | 1              | 2.7%             | 99.9%           |
+| 600    |  66,666         | 4 (0.01%)       | 8 (0.02%)      | 4.1%             | 149.9%          |
+| 800    |  50,000         | 17 (0.03%)      | 23 (0.04%)     | 5.5%             | 199.8%          |
+
+
+#### Base
```

Comment:
We should double check these figures against estimates from the base team cc @nira.